src/components/Mailbox.vue

```diff
@@ -386,17 +385,23 @@ export default {
 
 				break
 			case 'unseen':
-				if (this.hasSeenAcl()) {
+				logger.debug('marking as seen/unseen via shortcut')
```

Comment:
do you want to keep these debug logs?

Reply:
I've added a couple of debug statements, similar to the archive case, to make debugging easier. In the previous version, it was a bit hard to see if handleShortcut is even called. It might be a bit nicer to move the logic up and have a general debug statement like "shortcut x received." I don't have a strong opinion on this one, happy to remove them if you want me to.

Reply:
:shipit: